pages/news/2020-XX-XX-concrete-action.html

```diff
@@ -37,6 +37,7 @@
 
 As an entirely online community that allows anyone to participate with a pseudonym, we don't know the exact demographic makeup of our contributors.
 Our impression, however, is that the Mixxx contributor base is largely white, male, and either from the US or Western Europe.
+This is especially disappointing given that [dance music culture exists thanks to people of color](https://www.dukeupress.edu/Love-Saves-the-Day/).
```

Comment:
```suggestion
This is especially disappointing given that people of color played a major role in paving the way to DJing as an art form and [heavily influenced the dance music culture in the US and **elsewhere](https://www.dukeupress.edu/Love-Saves-the-Day/).**
```

Reply:
I don't think that's strong enough. people of color invented dance music and djing

Reply:
AFAIK Francis Grasso invented DJing, at least beatmatching and slip-cueing. But sure, almost all other major contributions I can think of were made by black artists.

Maybe make it a bit stronger? 
```suggestion
This is especially disappointing given that people of color played a crucial role in paving the way to DJing as an art form and [massively influenced the dance music culture in the US and elsewhere](https://www.dukeupress.edu/Love-Saves-the-Day/).
```

Reply:
crucial -> foundational?

Reply:
how about this?